pages/pipelines/secrets.md.erb

```diff
@@ -11,57 +11,56 @@ A best practice for secret storage is to use your own secrets storage service, s
 There are also various [Buildkite Plugins](/docs/plugins) that integrate reading
 and exposing secrets to your build steps using secrets storage services.
 
-## Storing secrets in environment hooks
+## Exporting secrets with environment hooks
 
-If you don't use a secrets storage service, the recommended place to store
-secrets is in an [`environment` hook](/docs/agent/v3/hooks#job-lifecycle-hooks).
-[Agent hooks](/docs/agent/v3/hooks#hook-locations-agent-hooks) are stored on
-your agent machine, are only accessible by you, and can conditionally expose
-secrets to your pipeline steps.
+If you don't use a secrets storage service, then you can use the Buildkite agent's `environment` hook to export secrets to a job.
 
-For example, say you had the following deploy [command step](/docs/pipelines/command-step) in a pipeline:
+The `environment` hook is a shell script that is sourced at the beginning of a job.
+It runs within the job's shell, so you can use it to conditionally run commands and export secrets within the job.
 
-```yml
-steps:
-  - command: scripts/trigger-github-deploy
-    key: trigger-github-deploy
-```
-{: codeblock-file="pipeline.yml"}
+By default, the `environment` hook file is stored in the agent's `hooks` directory.
+The path to this directory varies by platform; read the [installation instructions](/docs/agent/v3/installation) for the path on your platform.
+The path can also be overridden by the [`hooks-path`](/docs/agent/v3/hooks#hook-locations-agent-hooks) setting.
 
-This step runs the `scripts/trigger-github-deploy` script from the repository, and creates a GitHub Deployment using a [GitHub personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line). The script has the following source code:
+For example, suppose you want to expose a Test Analytics API token to a specific pipeline.
```

Comment:
I'd write around suppose, and in this context we can probably delete entirely? 

```suggestion
For example, to expose a Test Analytics API token to a specific pipeline.
```

Reply:
I tried to avoid a longer sentence and I guess it came out weird? Revised with b2e7ab36.